coderd/database/queries/provisionerdaemons.sql

```diff
@@ -28,6 +28,54 @@ JOIN
 WHERE
     provisioner_jobs.id = ANY(@provisioner_job_ids :: uuid[]);
 
+-- name: GetProvisionerDaemonsWithStatusByOrganization :many
+SELECT
+	sqlc.embed(pd),
+	CASE
+		WHEN pd.last_seen_at IS NULL OR pd.last_seen_at < (NOW() - (@stale_interval_ms::bigint || ' ms')::interval)
```

Comment:
> stale_interval_ms::bigint

is it more convenient than `::interval`?

Reply:
Since we don't have translation between e.g. duration and interval, this is the best way I know of to feed an integer unit affecting time into the query.

I'd much prefer something like [`pqinterval`](https://github.com/teepark/pqinterval) but since it's a lossy translation, not everyone is a fan.